src/game/raids/builders.ts

```diff
@@ -0,0 +1,10 @@
+import type { Boss } from 'game/raids';
+
+export function buildBoss(params: { id: number; name: string; background?: string }): Boss {
+  return {
+    ...params,
+    headshot: `https://assets.rpglogs.com/img/warcraft/bosses/${params.id}-icon.jpg`,
```

Comment:
Requesting review from @emallson mostly because of the change to use this rather than embed our own copy of the JPGs.

Reply:
ill double check but that is probably fine